spec/index.bs

```diff
@@ -3910,6 +3974,45 @@ dictionary GPURenderPassColorAttachmentDescriptor {
 };
 </script>
 
+<dl dfn-type=dict-member dfn-for=GPURenderPassColorAttachmentDescriptor>
+    : <dfn>attachment</dfn>
+    ::
+        A {{GPUTextureView}} describing the texture [=subresource=] the [=pipeline=] will output to
+        for this color attachment.
+
+    : <dfn>resolveTarget</dfn>
+    ::
+        A {{GPUTextureView}} describing the texture [=subresource=] that will receive the resolved
+        output for this color attachment if {{GPURenderPassColorAttachmentDescriptor/attachment}} is
+        multisampled.
+
+    : <dfn>loadValue</dfn>
+    ::
+        Issue: Describe this dictionary member
+
+    : <dfn>storeOp</dfn>
+    ::
+        Issue: Describe this dictionary member
+</dl>
+
+<div class=validusage dfn-for=GPURenderPassColorAttachmentDescriptor>
+    <dfn abstract-op>Valid Usage</dfn>
+
+    Given a {{GPURenderPassColorAttachmentDescriptor}} |this| the following validation rules
+    apply:
+
+    1. |this|.{{GPURenderPassColorAttachmentDescriptor/attachment}} must have a renderable color format.
+    1. If |this|.{{GPURenderPassColorAttachmentDescriptor/resolveTarget}} is not `null`:
+
+        1. |this|.{{GPURenderPassColorAttachmentDescriptor/attachment}} must be multisampled.
+        1. |this|.{{GPURenderPassColorAttachmentDescriptor/resolveTarget}} must not be multisampled.
+        1. Issue: Describe the remainder of resolveTarget validation
```

Comment:
Likewise I think we have these constraints:

 - The resolve texture view must be a single subresource.
 - It must have the same size as the attachment.
 - It must have the same format as the attachment.